azerothcore/azerothcore-wotlk
# Add support to calculate zoneId/areaId for creatures & gameojects

##### CHANGES PROPOSED:

-  Finishes the work of #1219 
- NOTE: THIS WILL SLOW UP THE STARTUP, DON'T USE FOR PUBLIC SERVERS, ONLY FOR DEVELOPING.

###### ISSUES ADDRESSED:


Closes #1225


##### TESTS PERFORMED:


Compiles and work as intended.

##### HOW TO TEST THE CHANGES:


Just open worldserver with the configs set to 1, then look at creature/gameobject table and see if zoneId and areaId are filled.

##### KNOWN ISSUES AND TODO LIST:



##### Target branch(es):

Master
